autoendpoint/src/extractors/subscription.rs

```diff
@@ -87,7 +87,7 @@ impl FromRequest for Subscription {
             if let Some(ref header) = vapid {
                 let sub = header
                     .vapid
-                    .sub()
+                    .insecure_sub()
                     .map_err(|e: VapidError| {
```

Comment:
`sub`/`insecure_sub` logs its errors, and here we emit a metric when they fail (in map_err). However we ultimately eat the error via `unwrap_or_default()` a few lines below: so I'm not seeing how this issue would lead to messages being rejected as invalid?

Reply:
Honestly, me neither. Part of the goal here is to just remove them as noise from our logs, as well as fix grabbing the right value.